config/profiler/run_profiler_integration_tests.sh

```diff
@@ -54,6 +54,9 @@ apt-get install sudo -qq -o=Dpkg::Use-Pty=0 >/dev/null 2>/dev/null # mask output
 sudo apt-get install unzip -qq -o=Dpkg::Use-Pty=0 >/dev/null 2>/dev/null # mask output
 pip install -q -r config/profiler/requirements.txt >/dev/null 2>/dev/null # mask output
 
+# Check for broken dependencies
+pip check
```

Comment:
This checks for broken dependencies but doesn't actually halt the build if there are broken dependencies.

Check the exit code of this call, print an appropriate message, and then call `exit 1`